components/usage-api/usage/v1/usage.proto

```diff
@@ -15,6 +15,66 @@ service UsageService {
 
     // GetCostCenter retrieves the spending limit with its associated attributionID
     rpc GetCostCenter(GetCostCenterRequest) returns (GetCostCenterResponse) {}
+
+    // Triggers reconciliation of usage with ledger implementation.
+    rpc ReconcileUsageWithLedger(ReconcileUsageWithLedgerRequest) returns (ReconcileUsageWithLedgerResponse) {}
```

Comment:
Nit: Would be great to find a simpler name (e.g. `reconcileUsageEntries`?) as we are likely deprecate and remove the other usage reconcilation method eventually.

Reply:
Once we deprecate the old one, we can actually rename this to `ReconcileUsage` as we control all the clients of the API. Happy to rename for now as well but I expect we'll rename this once more down the road.